dev-docs/publisher-api-reference.md

```diff
@@ -1474,14 +1467,38 @@ Additional information of these properties:
 | `endpoint` | Required | URL | Defines the auction endpoint for the Prebid Server cluster |
 | `syncEndpoint` | Required | URL | Defines the cookie_sync endpoint for the Prebid Server cluster |
 | `userSyncLimit` | Optional | Integer | Max number of userSync URLs that can be executed by Prebid Server cookie_sync per request.  If not defined, PBS will execute all userSync URLs included in the request. |
-| `adapterOptions` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server. |
+| `adapterOptions` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in request.ext.BIDDER. See the example above. |
+| `extPrebid` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in ext.prebid. See video-related example below. |
```

Comment:
in `adapterOptions` you specify `request` in `request.ext.BIDDER`

But in `extPrebid` you just say `in ext,prebid`

Should it be changed to `request.ext.prebid` in the latter for consistency?

Reply:
made it consistent. thanks @robertrmartinez